src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void noRepeatedUserLogin() {
```

Comment:
Давай будем придерживаться такой структуры для названий тестов -> methodUnderTest_state_expectedBehavior

```suggestion
    void register_userAlreadyExist_notOK() {
```